package.json

```diff
@@ -17,17 +17,17 @@
   "scripts": {
     "fix": "npm-run-all --continue-on-error \"fix:*\"",
     "fix:dependency-version-consistency": "check-dependency-version-consistency --fix .",
-    "fix:eslint": "eslint \"**/*\" --fix",
-    "fix:markdownlint": "markdownlint --fix \"**/*\"",
-    "fix:prettier": "prettier --cache --write --ignore-unknown \"**/*\"",
+    "fix:eslint": "eslint --fix \"**\"",
+    "fix:markdownlint": "markdownlint --dot --fix \"**\"",
```

Comment:
`--dot` includes files/folders with a dot (for example `.github`) — see [docs](https://www.npmjs.com/package/markdownlint-cli). We don't have markdowns in `.` folders but covering these files can be handy in the future.